### DISCUSSION TITLE
ONNX wired line file

### DISCUSSION THREAD

**Discussion by Holmes2002** [RapidAI/TableStructureRec#16] [Q&A]
Hi, I want to download wired line table file but I didn't see any link to download ONNX file for it. How can I download for this file and what kind of LORE model you converted to ONNX file ? If it was DLA-34 you can share way to convert to ONNX format for me ?
Thank you

**Comment by SWHL** [comment]
ONNX file has already been contained in the whl package. 
If you want to download them, you can download in this [link](https://github.com/RapidAI/TableStructureRec/releases/tag/v0.0.0)

I am not sure if it is the LORE model is DLA-34. The model I converted is [`damo/cv_desnet-transformer_table-structure-recognition-lore`](https://www.modelscope.cn/models/iic/cv_resnet-transformer_table-structure-recognition_lore)
